parser/utils.go

```diff
@@ -0,0 +1,48 @@
+package parser
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+)
+
+func parseDuration(split [][]byte, startIndex int, endIndex int, entryType string) (int, error) {
```

Comment:
In this case the non-capitalized function means not exported outside of package? (also for all other functions in this file and other files)?

Reply:
Yes, that's correct.

In this case, we don’t need visibility outside the `parser` package because the functionality is only relevant to the parsing logic.

Maybe I should rename the file to `parsing_utils.go`?

Reply:
Not needed to rename `utils.go`, it is already in the `parser` package.